### ISSUE TITLE
[BUG] Draw borders on top does not work

### ISSUE THREAD

**Issue by Zamrod** [p4535992/foundryvtt-token-factions#26]
**Module Version:** v0.4.7
As per #6 , this feature should be added but it does not appear in the settings at all and by default it draws underneath all tokens:

![image](https://github.com/p4535992/foundryvtt-token-factions/assets/12570777/77ade700-daaf-4322-b161-6decc55be55b)

**Comment by p4535992** [comment]
I believe the two tickets are separate ticket #6 refers to the fact that the border of the token was not "above" the frame, now it is as show in the image.

![22](https://i.imgur.com/rghKVDp.gif)

In the image you passed it looks like you want the "secondary border" of the frame (not the border of the token), to be above the image of the token ?

**Comment by Zamrod** [comment]
Correct. That's what I was requesting in the previous request. I'd like to essentially cover up the "border" built into the token image with the border from this module, since that used to be possible with the module.
